src/io/flutter/logging/FlutterConsoleLogManager.java

```diff
@@ -382,6 +396,23 @@ private void printDiagnosticsNodeProperty(ConsoleView console, String indent, Di
     }
   }
 
+  private void showDeepLinkNotification(DiagnosticsNode property) {
+    final Notification notification = new Notification(
+      FlutterMessages.FLUTTER_NOTIFICATION_GROUP_ID,
+      "Inspect this widget",
+      "Click below to inspect this exception-causing widget in Flutter DevTools",
```

Comment:
@csells @InMatrix for input on verbiage

@DanTup sent an example over to me that looked like this:
![Screen Shot 2021-01-25 at 2 14 15 PM](https://user-images.githubusercontent.com/43759233/105772884-ba55be00-5f17-11eb-8e17-2a51a2919d5d.png)

I think we should try to use consistent wording between the two IDEs, and I actually like how in this screenshot it just grabs the error message directly from the error, so it's consistent with console output as well.

For the button text, I like "Inspect widget"

Reply:
Updated to look like this:
<img width="468" alt="Screen Shot 2021-01-25 at 3 32 42 PM" src="https://user-images.githubusercontent.com/6379305/105779525-68b33080-5f23-11eb-8b82-df87a97f3efb.png">

There's a title as well as the button, let me know if any of you have preferences there.

Reply:
maybe change the title text to "Inspect error-causing widget"? 

this wording matches what the error message says. For example,

```
The relevant error-causing widget was:
  Column
  file:///Users/kenzieschmoll/develop/flutter_vignettes/vignettes/spending_tracker/lib/demo.dart:64:26
```

Reply:
Sure, like this?

<img width="378" alt="Screen Shot 2021-01-25 at 3 57 02 PM" src="https://user-images.githubusercontent.com/6379305/105780806-1de6e800-5f26-11eb-89a5-9ce401e45bc4.png">

I'll land with the current lame URL parsing and make updates in a later PR if there's a nicer way.

Reply:
I'm late to the party. The verbiage looks fine. I like icon in the VS Code version, which signals that this notification is related to an error. For the IntelliJ version, can we go without a title for the message to match the simplicity of the VS Code version? 

A related question: how are we handling multiple exceptions thrown in the same time? Will the user get a bunch of notifications at once? For example, this [Missing Material Widget error](https://github.com/InMatrix/flutter_error_studies/blob/master/lib/errors/missing_material_error.dart) will also generate a RenderFlex Overflow error from the red warning screen on device.

Reply:
I'm making another change for the length of time this notification is shown, so I can make the other adjustments there too. 

If there are multiple exceptions, they'll show up stacked like this:
<img width="389" alt="Screen Shot 2021-01-26 at 2 00 22 PM" src="https://user-images.githubusercontent.com/6379305/105913277-6d3e1e80-5fe1-11eb-9da1-595c9a66846c.png">

Reply:
Will the first error get displayed at the top of the stack or the bottom? The first error is usually relevant to the user's code in my experience.

Reply:
Hmm it's the most recent error that gets displayed. When "1 more" is clicked, there's a list shown of all the notifications with their timestamps:
![Screen Shot 2021-01-27 at 11 55 39 AM](https://user-images.githubusercontent.com/6379305/106046025-940a5c80-6096-11eb-9db0-94c9248e8c6c.png)

It may be the IntelliJ default to show the most recent at the top so that new notifications aren't missed. I can check whether this can be adjusted for a particular group of errors.

Reply:
Thanks for checking. I think we need to try it over a few example errors and determine what makes sense. Flutter framework often throws secondary exceptions down the pipeline caused by the primary error the user can fix in their own code. Maybe there are other filtering strategies we can employ.